.github/workflows/tests.yml

```diff
@@ -12,16 +12,27 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby: [2.6, 2.7, 3.0]
+        ruby: ['2.6', '2.7', '3.0', '3.1', '3.2']
```

Comment:
I'd like to drop Ruby < 3 support in the next major release.

Reply:
what about unsupported rails versions? those too?

Reply:
Yes, those too I think.